src/data/dfinsupp.lean

```diff
@@ -578,8 +578,14 @@ ext $ λ i, by simp only [neg_apply, mk_apply]; split_ifs; [refl, rw neg_zero]
   mk s (x - y) = mk s x - mk s y :=
 ext $ λ i, by simp only [sub_apply, mk_apply]; split_ifs; [refl, rw sub_zero]
 
-instance [Π i, add_group (β i)] {s : finset ι} : is_add_group_hom (@mk ι β _ _ s) :=
-{ map_add := λ _ _, mk_add }
+-- can I do this and get LaTeX in the docs?
+/-- If `s` is a subset of `ι` then `mk_add_group_hom s` is the canonical additive
+group homomorphism from $$\Prod_{i\in s}\beta\_i\to\Prod_{\mathtt{i : \io}}\beta_i$$.-/
```

Comment:
```suggestion
group homomorphism from $$\prod_{i\in s}\beta\_i\to\prod_{\mathtt{i : \iota}}\beta_i.$$-/
```
Judging from this page, you can: https://leanprover-community.github.io/mathlib_docs/analysis/mean_inequalities.html